src/genie/libs/parser/iosxe/tests/test_show_license_summary.py

```diff
@@ -0,0 +1,61 @@
+# Python
```

Comment:
Thank you for using folder based testing! Since we will have 2 tests for the same parser, please remove this file